### PR TITLE
Add N5193A lan streaming example

### PR DESCRIPTION
**Pull request by jeff922**
requirements.txt:
 removed non-run time requirements to simplify install
  
pdwBuilder.py:
 analog_bin_pdw_file_builder() - separated raw pdw section and end block for mode intuitive block separation
  
instruments.py:
 AnalogUXG():
  added bin_raw_pdw_block_builder() to build raw pdws without headers for direct lan streaming
  
examples.py:
 added analog_uxg_lan_stream_pdw_example() for direct lan streaming
 
 renamed analog_uxg_file_stream_pdw_example - added file to name to distinguish from lan streaming

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 PyArbTools: Keysight Signal Generator Control & Waveform Creation
 =================================================================
 
-*Current version: 2020.09.2*
+*Current version: 2021.01.15*
```
Comment (morgan-at-keysight) [inline README.rst]: Version should be `2021.01.1`. I have been breaking the versions down by month. So `2021.01.1` would be the first update in Jan 2021.

**Review by morgan-at-keysight — CHANGES_REQUESTED** [review]
@jeff-at-keysight these are all great changes. The only thing that needs attention is the version. There's a little comment there that explains it.

**Comment by jeff922** [comment]
I just reviewed and confirmed.

Thanks,

Jeff

**Comment by morgan-at-keysight** [comment]
@jeff-at-keysight if you've updated the code, can you add another commit to this pull request with the changes you made?

**Pull request merged**